src/main/java/com/epam/training/olha_haichenkova/task_3/service/VirtualMachineCreator.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.training.olha_haichenkova.task_3.service;
+
+import com.epam.training.olha_haichenkova.task_3.model.VirtualMachine;
+
+public class VirtualMachineCreator {
```

Comment:
To specific values for 'creator' class. If we need to change at least one parameter, we'll need one more creator with such approach